reboot_pending/reboot_pending.dsc.resource.json

```diff
@@ -16,11 +16,10 @@
   "schema": {
     "embedded": {
       "$schema": "https://json-schema.org/draft/2020-12/schema",
-      "type": "null",
+      "type": "object",
       "properties": {
         "rebootPending": {
-            "type": "boolean",
-            "readOnly": true
```

Comment:
We don't want to remove the `readOnly` keyword here, I think - it indicates that the property can only be retrieved by the resource, not managed, which is true in this case.